test/es6ToJSONTest.js

```diff
@@ -113,32 +111,67 @@ describe('#ToJSON', () => {
     const RecursiveEntry = importResult('shr/simple/RecursiveEntry');
     it('should serialize a JSON instance', () => {
       // This one is special cased because you're working with recursive entries
-      const json = getJSON('RecursiveEntry');
+      const json = context.getJSON('RecursiveEntry');
       const entry = RecursiveEntry.fromJSON(json);
       expect(entry).instanceOf(RecursiveEntry);
       let gen_json = entry.toJSON();
-      validateJSON('RecursiveEntry', gen_json);
+      context.validateJSON('RecursiveEntry', gen_json);
       expect(gen_json).to.eql(json);
       
       // Recursive child 1
       const child1 = entry.recursiveEntry[0];
       expect(child1).instanceOf(RecursiveEntry);
       let child1_json = child1.toJSON();
-      validateJSON('RecursiveEntry', child1_json);
+      context.validateJSON('RecursiveEntry', child1_json);
       expect(gen_json).to.eql(json);
```

Comment:
This was apparently in the original code, but just noticing now, neither `gen_json` nor `json` have been modified since last time we tested `expect(gen_json).to.eql(json)`.  What is the purpose of testing it again?